.github/labeler.yml

```diff
@@ -0,0 +1,25 @@
+# https://github.com/actions/labeler#common-examples
+# Adapted from https://github.com/rapidsai/cudf/blob/main/.github/CODEOWNERS
+# Labels culled from https://github.com/rapidsai/cugraph/labels
+
+python:
+  - python/**
+  - notebooks/**
```

Comment:
@rlratzel should the `benchmark` directory be there?

Reply:
yes, the benchmark dir is python code we run nightly and as part of CI (in test-only mode) so we'd want that added as well.